portfolio/src/main/webapp/style.css

```diff
@@ -83,26 +87,42 @@ body{
   margin-bottom:10px
 }
 
+
 .cardSecondary{
 }
 
 .cardLongDesc{
   display: none;
+  opacity: 0.0;
 }
 
 .openCard{
-  width:70vw;
-  min-height: 300px;
+  /*min-height: 300px;*/
 }
 
 .openCard .cardImg{
-  display: none;
+  height: 0px;
+  /*animation: openCardImg 0.2s linear 0s forwards;*/
 } 
 
 .openCard .cardLongDesc{
   display: block;
+  animation: openCardLongDesc 0.2s ease-in 0.5s forwards;
 } 
 
+@keyframes openCardImg{
+  to{height: 0px;}
+}
+
+@keyframes openCardLongDesc{
+  to{opacity: 1.0}
+}
+
+
+/*.cardContainer:has(> div) .card{
+  display: none;
+}*/
```

Comment:
recommend removing code that is not used anymore. If you ever want to go back to them you should find them in the commit history.

Reply:
Same for some comments above.